toolkits/global/packages/global-buttons/scss/10-settings/buttons.scss

```diff
@@ -0,0 +1,69 @@
+/**
+ *  Button settings
+ *
+ */
+
+$button-base--text-align: left;
```

Comment:
I think the differentiation between base and shared is confusing. You'd have to go into the code to understand which settings to change for your brand.
E.g. shared means it's shared between regular and circle? Which then suggests that other buttons don't use that?

It would read better IMO if it was just `$button--padding`.
Then _if_ we need a circle button, something like `$button--circle-padding: $button--padding` (allows the brand to configure but sets a single default).

Reply:
oh, also I think buttons should default to center text aligned